library/src/main/java/com/google/maps/android/clustering/view/DefaultClusterRenderer.java

```diff
@@ -79,6 +81,7 @@
     private final ClusterManager<T> mClusterManager;
     private final float mDensity;
     private boolean mAnimate;
+    private final Executor mExecutor = Executors.newSingleThreadExecutor();
```

Comment:
Any reason this should be single threaded vs. a larger thread pool? Making this single threaded changes the current behavior/performance.

Reply:
Yes, as noted by @barbeau https://github.com/googlemaps/android-maps-utils/pull/601#discussion_r369624658 when reviewing the original PR, the implementation is effectively single threaded already, which is why I chose to use a single thread executor. There's only ever one render task in flight and potentially one queued up to run next.

Reply:
Got it! That wasn't immediately obvious to me so this is definitely a nice improvement 👍